target/linux/ramips/image/mt7621.mk

```diff
@@ -221,6 +221,20 @@ define Device/zbt-wg3526
 endef
 TARGET_DEVICES += zbt-wg3526
 
+define Device/hc5962
+  DTS := HC5962
+  BLOCKSIZE := 128KiB
+#  IMAGES := sysupgrade.bin
+  PAGESIZE := 2048
+  KERNEL_SIZE := 2097152
+#  FILESYSTEMS := squashfs
+  UBINIZE_OPTS := -E 5
+  IMAGE/sysupgrade.bin := append-kernel | pad-to $$(KERNEL_SIZE) | append-ubi | check-size $$$$(ralink_default_fw_size_32M)
+  DEVICE_TITLE := HiWiFi HC5962
+  DEVICE_PACKAGES := kmod-usb3 kmod-usb-ledtrig-usbport kmod-mt76
```

Comment:
You add "kmod-usb-ledtrig-usbport" module, but you not defined any led as usbport.